en/docs/learn/outbound-provisioning-with-salesforce.md

```diff
@@ -281,14 +281,14 @@ Provider.
     | LanguageLocaleKey                  | `               http://wso2.org/claims/dob              `                      |
     | LastName                           | `               http://wso2.org/claims/lastname              `                 |
     | LocaleSidKey                       | `               http://wso2.org/claims/primaryChallengeQuestion              ` |
-    | ProfileId                          | `               http://wso2.org/claims/role              `                     |
+    | ProfileId                          | `               None             `                                             |
     | TimeZoneSidKey                     | `               http://wso2.org/claims/challengeQuestion1              `       |
     | UserPermissionsCallCenterAutoLogin | `               http://wso2.org/claims/telephone              `                |
     | UserPermissionsMarketingUser       | `               http://wso2.org/claims/mobile              `                   |
     | UserPermissionsOfflineUser         | `               http://wso2.org/claims/country              `                  |
     | Username                           | `               http://wso2.org/claims/emailaddress              `             |
-
-    ![add-claim-mapping](../assets/img/using-wso2-identity-server/add-claim-mapping.png) 
+    
+    ![add-claim-mapping](https://user-images.githubusercontent.com/28682701/174729411-c9c201b1-a376-4c73-8d61-cd230858a923.png)
```

Comment:
Images need to be added into the doc-is repo and pointed from the md file

Reply:
Did the changes. Thank you!